### PR TITLE
added pyramid_tm

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
I could have sworn I already did this, but can't find it now, so here we go, maybe again :-)

@conda-forge/help-python: see #14020: one more Pyramid add on.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ChrisBarker-NOAA** [comment]
@JamesMakela-NOAA: Are you OK with being a maintainer? Please post in the affirmative on this issues.

**Comment by JamesMakela-NOAA** [comment]
> @JamesMakela-NOAA: Are you OK with being a maintainer? Please post in the affirmative on this issues.

Yes, I am ok with being a maintainer.

File: recipes/pyramid_tm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: 5fd6d4ac9181a65ec54e5b280229ed6d8b3ed6a8f5a0bcff05c572751f086533
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/pyramid_tm/meta.yaml:19]:
```suggestion
    - python >=3.6
```

**Comment by ChrisBarker-NOAA** [comment]
Thanks!

**Pull request merged**